### PR TITLE
rails generate hyrax:collection

### PR DESCRIPTION
**Pull request by darrenleeweber**
Double checking the latest hyrax generators - hyrax:collection in this PR.

@projecthydra-labs/hyrax-code-reviewers

### CODE REVIEW

File: app/models/collection.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
-# Generated by curation_concerns:models:install
+# Generated by hyrax:models
```
Comment (jcoyne) [inline app/models/collection.rb]: Technically this was generated by curation_concerns (before we moved to hyrax). But I don't feel strongly about this.

**Comment by jcoyne** [comment]
@darrenleeweber why is this marked `[WIP]`?  Is there more to come?

**Comment by darrenleeweber** [comment]
I've marked all the `rails generate` PRs as [WIP], pending advice on whether they are good changes or not.  There's nothing more to add here.

**Comment by jcoyne** [comment]
@darrenleeweber there is no need to mark something WIP unless you are planning to add more to it.  If you don't mark it, we will consider it ready for review.

**Pull request merged**